packages/cli/test/functional/test_site/testPageVariablesInInclude.md

```diff
@@ -1,3 +1,3 @@
 **Outer Page Variable Should Not Leak Into Inner Pages**
 
-{{ nested_page_variable or "Outer Page Variable Should Not Leak Into Inner Pages" }}
+{{ outerNunjucksVariable or "Outer Page Variable Should Not Leak Into Inner Pages" }}
```

Comment:
Note: updating this because it appears to be outdated. we used to have `nested_page_variable` defined in `index.md` but that has been deleted for a long time. See https://github.com/yucheng11122017/markbind/commit/2ddf00eca8300f390a3fe2650e2d6a2177b85c69
![image](https://user-images.githubusercontent.com/41845017/235271755-1a13b7dc-d60c-491a-8363-0e06c5e34871.png)